debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/MongoDataConverterTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb;
+
+import java.io.IOException;
+import java.util.Map.Entry;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.bson.BsonDocument;
+import org.bson.BsonValue;
+import org.junit.Assert;
+import org.junit.Test;
+
+import io.debezium.connector.mongodb.transforms.MongoDataConverter;
+
+public class MongoDataConverterTest {
+
+    public MongoDataConverterTest() throws IOException {
+}
+    ClassLoader classLoader = getClass().getClassLoader();
+    String record = IOUtils.toString(classLoader.getResourceAsStream("restaurants6.json"));
```

Comment:
I'd prefer to not introduce the Commons IO dependency just for that. It can easily be done with standard Java APIs, e.g. see the answer based on `Scanner` here: https://stackoverflow.com/questions/309424/read-convert-an-inputstream-to-a-string.